### PR TITLE
feat: improve DX

### PR DESCRIPTION
**Pull request by pi0**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

- Remove showReady from Core/Server
- Added a fancy banner inside CLI
- Listener refactored out of the Server
- Disable clearConsole
- Migrate to WebpackBar 3
-  Pause logs while building
- Wrap all console logs into consola

Fancy badge:

image

Extend support: (Via options.cli.badgeMessages)

image

Error stacks:

image

Colorized commands:

image

image

Fancier fatal error handler:

image

More informative messages:

image

No more clear screens. Each dev-reload has its own log:

image

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/server/src/listener.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    // Call server.listen
+    this.server = await new Promise((resolve, reject) => {
+      const s = this._server.listen(listenArgs, error => error ? reject(error) : resolve(s))
+    })
+
+    // Enable destroy support
+    enableDestroy(this.server)
+    pify(this.server.destroy)
```
Comment (clarkdo) [inline packages/server/src/listener.js]: `server-destroy` is a wrapper a `server.close` but not supports `async`, I think this part cannot work well, if you want I can create async version `server-destroy`

Comment (pi0) [reply packages/server/src/listener.js]: It should pass `cb` (which is generated by pify for making promise) directly into `close` call so works :)

Comment (clarkdo) [reply packages/server/src/listener.js]: Right, ignore me

Comment (clarkdo) [reply packages/server/src/listener.js]: Since lots of codes in nuxt are using `promisify`, are we going to replace ?

**Pull request merged**